addons/website/static/src/components/navbar/navbar.js

```diff
@@ -44,11 +44,18 @@ patch(NavBar.prototype, 'website_navbar', {
         useBus(websiteSystrayRegistry, 'CONTENT-UPDATED', renderAndAdapt);
     },
 
+    get isWebsiteSystray() {
```

Comment:
Seems strange that a "navbar" object has a function "isWebsiteSystray"... should not it be something like "shouldDisplayWebsiteSystray", something like that?

Reply:
Changed